app/src/main/res/values-fr/strings.xml

```diff
@@ -43,7 +43,7 @@
 
     <!-- Accessibility -->
     <string name="a11y_plant_item_image">Image de plante</string>
-    <string name="gallery_title">(translate me) Photos by Unsplash</string>
-    <string name="gallery_content_description">(translate me) Navigate to gallery screen</string>
+    <string name="gallery_title">Photos par Unsplash</string>
+    <string name="gallery_content_description">Naviguer vers l'écran de la galerie</string>
```

Comment:
```suggestion
    <string name="gallery_content_description">Naviguer vers l\'écran de la galerie</string>
```

Reply:
This is causing the build issue. You need to fix it first before applying. Thanks!